src/System.Numerics.Vectors/src/System/Numerics/Quaternion.cs

```diff
@@ -155,9 +155,9 @@ public static Quaternion Inverse(Quaternion value)
         }
 
         /// <summary>
-        /// Creates a Quaternion from a vector and an angle to rotate about the vector.
+        /// Creates a Quaternion from a unit vector axis and an angle to rotate about the vector.
         /// </summary>
-        /// <param name="axis">The vector to rotate around.</param>
+        /// <param name="axis">The unit vector to rotate around. This must be a vector of length 1.</param>
```

Comment:
It is not verified anywhere, if you pass in a vector that hasn't been normalized, you will probably not get the result you expect.

Perhaps "the given vector must be normalized prior to calling this function" would be better?